miio/vacuumcontainers.py

```diff
@@ -324,7 +324,7 @@ def sensor_dirty(self) -> timedelta:
 
     @property
     def sensor_dirty_left(self) -> timedelta:
-        return self.side_brush_total - self.sensor_dirty
+        return self.sensor_dirty_total - self.sensor_dirty
```

Comment:
This is still wrong :-) Also, please update `vacuum_cli.py` accordingly to display the new property.